saleor/graphql/webhook/subscription_types.py

```diff
@@ -214,6 +214,18 @@ def resolve_new_email(root, _info: ResolveInfo):
         return data["new_email"]
 
 
+class AccountEmailChanged(SubscriptionObjectType, AccountOperationBase):
+    new_email = graphene.String(
+        description="The new email address the user wants to change to.",
```

Comment:
To be precise, this is the new email address that the user has already changed to, at the point when the event is fired.

Reply:
ah right - I thought it is before the change, but change is already made but this is just a confirmation - so @ that step email is changed already but not confirmed yet